src/ui/deleteAccount/ViewDeleteAccountRequest.qml

```diff
@@ -7,25 +7,111 @@ import QtQuick.Layouts 1.14
 
 import Mozilla.VPN 1.0
 import components 0.1
+import components.forms 0.1
 import components.inAppAuth 0.1
 
-Item {
-   // TODO
-   // This view shows the delete request. There is the list of attached clients.
+VPNInAppAuthenticationBase {
+    id: viewDeleteAccount
 
-    Text {
-        id: msg
-        text: "A" + VPNAuthInApp.attachedClients
-        anchors.top: parent.top
+    property var checkboxData: [
+        {
+            labelText: "Paid subscriptions", // TODO: Needs content
```

Comment:
This needs to be localized if we want to keep the checkbox headers: UX and content are taking a look at this.